libs/SalesforceSDKCore/SalesforceSDKCore/Classes/NativeLoginManagerInternal.swift

```diff
@@ -39,25 +39,70 @@ let maximumPasswordLengthInBytes = 16000
 /// See https://developer.apple.com/documentation/swift/importing-swift-into-objective-c#Import-Code-Within-a-Framework-Target
 @objc(SFNativeLoginManagerInternal)
 public class NativeLoginManagerInternal: NSObject, NativeLoginManager {
+    
+    static let errorDomain = "com.salesforce.security.nativeLoginException"
```

Comment:
What's this used for?

Reply:
...Absolutely _nothin'_ since I made the error handling smoother in a newer commit during self-review.  I'll remove this.  Good catch!